deploy-manage/deploy/cloud-enterprise/ece-install-offline-images.md

```diff
@@ -65,17 +65,34 @@ Enterprise Search is not available in versions 9.0+.
 ::::{dropdown} Expand to view the full list
 | Required downloads | Minimum required ECE version |
 | --- | --- |
+| [{{es}}, {{kib}}, APM, and Enterprise Search stack pack: 9.0.1](https://download.elastic.co/cloud-enterprise/versions/9.0.1.zip) | ECE 4.0.0 |
```

Comment:
Should we also update the references to `8.18.0` and `9.0.0` in [ece-install-offline-no-registry.md](https://github.com/elastic/docs-content/blob/6e49ca70d605283016e0335ea5ace37fe6070fcd/deploy-manage/deploy/cloud-enterprise/ece-install-offline-no-registry.md) and [ece-install-offline-with-registry.md](https://github.com/elastic/docs-content/blob/6e49ca70d605283016e0335ea5ace37fe6070fcd/deploy-manage/deploy/cloud-enterprise/ece-install-offline-with-registry.md)?

Reply:
I think Nick is right, we should only update that when we release ECE. The docs should match whatever comes pre-installed with ECE because the customer will have to download the exact images that ECE will later require when installing.

Reply:
The files changed here are the right ones, they map to the download page where customers can download additional stackpacks.

Reply:
That makes sense, thank you both for the clarification 🙂